main.go

```diff
@@ -151,6 +158,9 @@ func main() {
 				if argv.SQLNullStr {
 					g.WithSQLNullStr()
 				}
+				if templates := []string(argv.TemplateFileNames.Value()); len(templates) > 0 {
+					g.WithTemplates(templates...)
```

Comment:
Let's keep the glob resolution out here, before going into the generator, so that it _only_ has to worry about known files, rather than expanding others.

If you could, take the globbing code below, put it into a function that takes in a filename, and returns an array of strings and an error so that we can use it for both input files as well as template files.